source/getting-started/installation-raspberry-pi-image.markdown

```diff
@@ -36,21 +36,97 @@ Some extra tips:
+
+Connect to the Raspberry Pi over ssh. Default user name is `pi` and password is `raspberry`.
+Linux and Mac OS users execute the following command in a terminal.
+```bash
+$ ssh pi@ip-address-of-pi
+```
+Windows users start [Putty][ssh-putty], enter the IP address of the Raspberry Pi in the *Host name* field and port 22 in the *Port* field. Then click *Open* and a terminal window will open. Enter the credentials. Default user name is `pi` and password is `raspberry`.
+
+#### {% linkable_title Start/Stop/Restart Home Assistant on HaSSbian %}
+Log in as the `pi` account account and execute the following commands:
+```bash
+sudo systemctl stop home-assistant@homeassistant.service 
+```
+Replace `stop` with `start` or `restart` to get the desired functionality.
+
+#### {% linkable_title Update Home Assistant on HASSbian %}
+Log in as the `pi` account and execute the following commands:
+```bash
+sudo systemctl stop home-assistant@homeassistant.service 
+sudo su -s /bin/bash homeassistant
+source /srv/homeassistant/bin/activate
+pip3 install --upgrade homeassistant
+exit
+sudo systemctl start home-assistant@homeassistant.service
+```
+This will in order do the following:
+- Stop the Home Assistant service running on HASSbian
+- Open a shell as the `homeassistant` user running the Homeassistant service and that has ownership over the Home Assistant installation.
+- Change into the virtual Python environment at `/src/homeassistant/` containing the Home Assistant installation.
+- Upgrade the Home Assistant installation to the latest release.
+- Exit the shell and return to the `pi` user.
+- Restart the Home Assistant service.
+
+#### {% linkable_title Manually launch Home Assistant on HASSbian %}
+Log in as the `pi` account and execute the following commands:
+```bash
+sudo su -s /bin/bash homeassistant
+source /srv/homeassistant/bin/activate
+hass
+```
+This will start Home Assistant in your shell and output anything that ends up in the log and more into the console. This will fail if the Home Assistant service is already running so don't forget to [stop][stop-homeassistant] it first.
+
+#### {% linkable_title Check your configuration on HASSbian %}
+Log in as the `pi` account and execute the following commands:
+```bash
+sudo su -s /bin/bash homeassistant
+source /srv/homeassistant/bin/activate
+hass --script check_config
+```
+This will output any errors in your configuration files to console.
+
+#### {% linkable_title Read the Home Assistant log file on HASSbian %}
+Log in as the `pi` account and execute the following commands:
+```bash
+sudo su -s /bin/bash homeassistant
+cd /home/homeassistant/.homeassistant
+nano homeassistant.log
+```
+This will in order do the following:
+- Open a shell as the `homeassistant` user.
+- Change directory to the Home Assistant configuration directory.
+- Open the log file in the nano editor.
+
+#### {% linkable_title Edit the Home Assistant configuration on HASSbian %}
```

Comment:
I am guessing that users can use WinSCP (or equivalent) to edit the config files and it may be good to add the same. The instructions are there on the AIO page.